pkg/projects/resource_project.go

```diff
@@ -177,22 +198,38 @@ func projectResource() *schema.Resource {
 			})
 		}
 
+		if users != nil {
```

Comment:
I am much more of a fan of having an empty slice than a nil slice. `nil` allows for `NilPointerExceptions` where as an empty slice is always safe